### PR TITLE
[12.0][MIG] purchase_order_approved

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

### CODE REVIEW

**Comment by MiquelRForgeFlow** [comment]
@lreficent

File: purchase_order_approved/models/res_config.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    purchase_approve_active = fields.Boolean(
+        related='company_id.purchase_approve_active',
+        string="State 'Approved' in Purchase Orders",
```
Comment (AdriaGForgeFlow) [inline purchase_order_approved/models/res_config.py:12]: Needs `readonly=False` so it can be edited from the Settings Menu

**Review by AdriaGForgeFlow — CHANGES_REQUESTED** [review]
Looks good just small comment

File: purchase_order_approved/models/res_config.py (hunk 1 above)
Comment (MiquelRForgeFlow) [reply purchase_order_approved/models/res_config.py:12]: done

**Review by AdriaGForgeFlow — CHANGES_REQUESTED** [review]
Deliveries are created even if the Purchase Order is in Approved state:

image

**Review by AdriaGForgeFlow — APPROVED** [review]
Fixed now :+1:

Tested on runbot.

**Review by LoisRForgeFlow — APPROVED** [review]
Functional and code review :+1:

**Review by nikul-serpentcs — APPROVED** [review]
If you increase `Codecov`  then Better Otherwise Code Review LGTM :+1:

**Comment by nikul-serpentcs** [comment]
ping @pedrobaeza @jbeficent

**Pull request closed without merging**